File: Python/Wycieraczka.py
import math
a, r = map(int, input().split())
n = int(input())
insects = []
for _ in range(n):
    x, y = map(int, input().split())
    insects.append((x, y))
notDeleted = 0
for x, y in insects:
    if math.sqrt((x - a)**2 + y**2) > r:
        notDeleted += 1
print(notDeleted)

File: C++/Wycieraczka.cpp
#include <cmath>
#include <vector>
#include <iostream>

int main()
{
    int a, r, n, notDeleted = 0;
    std::cin >> a >> r;
    std::cin >> n;
    std::vector<std::pair<int, int>> insects(n);
    for (int i = 0; i < n; ++i) 
       std::cin >> insects[i].first >> insects[i].second;
    for (const auto& insect : insects) 
        if (sqrt(pow(insect.first - a, 2) + pow(insect.second, 2)) > r) 
            notDeleted++;
    std::cout << notDeleted;
    return 0;
}